src/main/java/javaClasses/homework_4/Lilit_Manukyan/Developer.java

```diff
@@ -0,0 +1,27 @@
+package javaClasses.homework_4.Lilit_Manukyan;
+
+public class Developer extends Staff {
+    private String techstack;
+    public Developer(String name, String lastName, String techstack) {
+        super(name, lastName);
+        this.techstack = techstack;
+    }
+
+    public String getTechstack() {
+        return techstack;
+    }
+
+    public String setTechstack(String techstack) {
```

Comment:
Your setter does the same as the getter. I got the idea, but please change the implementation. And your setter should be returning nothing /void type/, please change that as well.